### PR TITLE
Keyboard helpers (closes #431)

### LINKED ISSUE #431: Keyboard helpers
Think about separate module with common key codes, like enter, arrow keys, etc

### PR DESCRIPTION
**Pull request by narqo**
\cc @dfilatov 

Should I write specs for this? ;)

### CODE REVIEW

**Comment by dfilatov** [comment]
I think `keyboard` is too general name for module that provides key codes. Perhaps, it would be better if we have `keyboard__keys` module. It also ables us to remove `KEY_` prefix from constants.

``` js
modules.define('keyboard__keys', function(provide) {

provide(/** @exports */{
    BACKSPACE : 8,
    TAB : 9,
    ...
});

});
```

**Comment by narqo** [comment]
Ok, I thought about `keyboard__codes` but decided that it will be too complicated.

The other question is, should we have API for this kind of tasks (we have a huge amount of this crap internally, mainly for a11y staff to work):

``` js
this.bindTo('action', 'keyup', function(e) {
  if(e.shiftKey || e.ctrlKey || e.altKey || e.metaKey) {
    return;
  }
  // ...
})
```

**Comment by dfilatov** [comment]
keyboard__codes looks better than keyboard__keys :)

File: common.blocks/keyboard/__keys/keyboard__keys.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * @module keyboard__keys
+ */
+modules.define('keyboard__keys', function(provide) {
+
+provide(/** @exports */{
+    BACKSPACE : 8,
```
Comment (persidskiy) [inline common.blocks/keyboard/__keys/keyboard__keys.js]: Why do you use uppercase for object keys? Maybe using lowercase will be quite?

Comment (narqo) [inline common.blocks/keyboard/__keys/keyboard__keys.js]: It's good practice to use uppercase for constants. Actually we always write our code in this way: https://github.com/bem/bem-core/blob/v2/common.blocks/i-bem/__internal/i-bem__internal.vanilla.js#L57-L58

**Comment by narqo** [comment]
> keyboard__codes looks better than keyboard__keys :)

For me the main question is how to call required variable for define/require's callback? 

``` js
modules.require(
  ['keyboard', 'keyboard__keys', 'keyboard__codes'],
  function(keyboard, keys, kbCodes) {

keyboard.ENTER; 
keys.ENTER; 
kbCodes.ENTER;

});
```

**Comment by dfilatov** [comment]
`keyCodes.ENTER`?

**Comment by narqo** [comment]
> `keyCodes.ENTER?`

Why not :) :up:

File: common.blocks/keyboard/__keys/keyboard__keys.js (hunk 1 above)
Comment (persidskiy) [inline common.blocks/keyboard/__keys/keyboard__keys.js]: Ok

**Comment by dfilatov** [comment]
:shipit:

**Pull request merged**